sqlserver/datadog_checks/sqlserver/connection.py

```diff
@@ -110,9 +110,11 @@ class Connection(object):
     DEFAULT_DATABASE = 'master'
     DEFAULT_DRIVER = 'SQL Server'
     DEFAULT_DB_KEY = 'database'
+    DEFAULT_SQLSERVER_VERSION = "2019"
+    SQLSERVER_2014 = 2014
     PROC_GUARD_DB_KEY = 'proc_only_if_database'
 
-    valid_adoproviders = ['SQLOLEDB', 'MSOLEDBSQL', 'SQLNCLI11']
+    valid_adoproviders = ['SQLOLEDB', 'MSOLEDBSQL', 'MSOLEDBSQL19', 'SQLNCLI11']
```

Comment:
small cleanup nit: can convert to an immutable type to make this safer. Either a tuple or frozenset